cfme/provisioning.py

```diff
@@ -35,6 +40,137 @@ def select_security_group(sg):
     sel.sleep(1)
 
 
+class ProvisioningForm(BaseLoggedInPage):
+    @View.nested
+    class request(Tab):  # noqa
+        TAB_NAME = 'Request'
+        email = Input('requester__owner_email')
```

Comment:
@mfalesni @izapolsk  When I try to fill the form with a nested dictionary of values, this field's fill raises an exception at widget.TextInput.fill(). self.value triggers the exception for NoSuchElement.

While debugging, the form was loaded, and manually searching for the locator stored in the TextInput object showed that the xpath was correct.